README.md

```diff
@@ -4,11 +4,20 @@
 
 ### Disclaimer: work in progress and not ready for production use. 
 
-kube-bind is a prototype project with the goal to establish a new extension model for Kubernetes clusters:
+You are invited to [contribute](#contributing)!
 
-- APIs should be bindable into a cluster and operated by a service provider
-- these APIs should not require (custom) controllers/operators run locally in the consuming cluster
-- only a single vendor-neutral, OpenSource agent should be required.
+## What is it?
+
+kube-bind is a prototype project that aims to provide better support for service providers and consumers that reside in distinct Kubernetes clusters.
+
+- A service provider defines its API in terms of CRDs and associated permission claims/limitations, and exports it for use from other clusters.
+- Service consumers identify the services they want to consume.
```

Comment:
`Consumer` itself sounds better than `Service consumer` to me. You might have different thought.

Reply:
okay, happy with it